backup/src/main/java/io/camunda/zeebe/backup/management/BackupService.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.backup.management;
+
+import io.camunda.zeebe.backup.api.BackupManager;
+import io.camunda.zeebe.backup.api.BackupStatus;
+import io.camunda.zeebe.scheduler.Actor;
+import io.camunda.zeebe.scheduler.future.ActorFuture;
+import io.camunda.zeebe.scheduler.future.CompletableActorFuture;
+
+/** Backup manager that takes and manages backup asynchronously */
+public class BackupService extends Actor implements BackupManager {
+
+  @Override
+  public void takeBackup(final long checkpointId, final long checkpointPosition) {
+    // Will be implemented later
+  }
+
+  @Override
+  public ActorFuture<BackupStatus> getBackupStatus(final long checkpointId) {
+    return CompletableActorFuture.completedExceptionally(new RuntimeException("Not implemented"));
```

Comment:
:upside_down_face: there's `UnsupportedOperationException`, which we could also throw in `takeBackup`.